script/hassfest/translations.py

```diff
@@ -329,6 +329,15 @@ def gen_strings_schema(config: Config, integration: Integration) -> vol.Schema:
                 flow_title=UNDEFINED,
                 require_step_title=False,
             ),
+            vol.Optional("preview_features"): cv.schema_with_slug_keys(
+                {
+                    vol.Required("name"): translation_value_validator,
+                    vol.Required("description"): translation_value_validator,
+                    vol.Optional("enable_confirmation"): translation_value_validator,
+                    vol.Optional("disable_confirmation"): translation_value_validator,
```

Comment:
Why are these options set here? Does this mean we have to update hassfest whenever an integration adds a new preview feature?

Reply:
Ok, but having to update hassfest for every preview feature seems like an antipattern. I'd expect the integration that wants to add a preview feature should be responsible for the translation strings of its preview feature and those strings should just be hosted in strings.json of the integration. No general validation change of translations here in hassfest should have to be done for every preview feature.